pandas/core/generic.py

```diff
@@ -7708,6 +7708,8 @@ def resample(
         ----------
         rule : DateOffset, Timedelta or str
             The offset string or object representing target conversion.
+            The str value refers to frequency string.
```

Comment:
this line is redundant. you can add a link the documentation if you'd like; i think we have this link elsewhere.